catalystwan/workflows/config_migration.py

```diff
@@ -308,7 +307,7 @@ def transform(ux1: UX1Config) -> ConfigTransformResult:
             policy_parcel = convert_policy_list(policy_list)
             header = TransformHeader(type=policy_parcel._get_parcel_type(), origin=policy_list.list_id)
             ux2.profile_parcels.append(TransformedParcel(header=header, parcel=policy_parcel))
-        except PolicyListConversionError as e:
+        except CatalystwanConverterCantConvertException as e:
```

Comment:
```suggestion
        except CatalystwanConverterCantConvertException as e:
              exception_message = "Something about policy"
              logger.warning(exception_message)
              transform_result.add_failed_conversion_parcel(
                  exception_message=exception_message,
                  policy=policy_list,
              )
```

Also maybe we should catch Pydantic ` ValidationError ` just in case?

Reply:
exception message is in first argument passed to exception constructor and it is printed in logger warning below. Yeah we might catch validation error also